modules/update-etcd-backup.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="update-etcd-backup_{context}"]
+= etcd backups before cluster updates
+
+etcd backups record the state of your cluster and all of its resource objects.
+You can use backups to attempt restoring the state of a cluster in disaster scenarios where you cannot recover a cluster in its currently dysfunctional state.
+
+In the context of updates, you can attempt an etcd restoration of the cluster in a case where an update introduced catastrophic conditions that cannot be fixed without reverting to the cluster version prior to the update.
```

Comment:
```suggestion
In the context of updates, you can attempt an etcd restoration of the cluster in a case where an update introduced catastrophic conditions that cannot be fixed without reverting to the cluster version prior to the update.
```
I'm finding this sentence difficult to parse out. Maybe try to simplify it?

Reply:
How about this:
```suggestion
In the context of updates, you can attempt an etcd restoration of the cluster if an update introduced catastrophic conditions that cannot be fixed without reverting to the previous cluster version.
```

Reply:
Yes, that's better!